packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -488,6 +494,14 @@ export default {
       type: Boolean,
       default: true,
     },
+
+    /**
+     * Sending this prop to product side as a temporary measure to use a slot for emoji & giphy picker.
+     */
+    isNewMessageInput: {
```

Comment:
Do we really need this prop considering we already have `showEmojiPicker`?

Reply:
I think "showEmojiPicker" is used to decide if we want to show the emoji picker or not. The new prop is used to decide if we want to show the Dialtone emoji selector or not. I don't know if there's a way to check if a slot was passed so in that case we don't need a new prop.
If slot was sent -> rendering slot
If not -> rendering dialtone emoji picker

Reply:
If you want the Dialtone emoji picker, you could use it and leave the new slot empty. If you want to use the new slot, you could set `showEmojiPicker` to false. That's why I don't see much use for a new prop for this.

Reply:
okey, yeah, makes sense now